go-controller/pkg/ovn/endpoints.go

```diff
@@ -88,8 +88,6 @@ func (ovn *Controller) AddEndpoints(ep *kapi.Endpoints) error {
 				klog.Errorf("Error in creating Cluster IP for svc %s, target port: %d - %v\n", svc.Name, lbEps.Port, err)
 				continue
 			}
-			vip := util.JoinHostPortInt32(svc.Spec.ClusterIP, svcPort.Port)
-			ovn.AddServiceVIPToName(vip, svcPort.Protocol, svc.Namespace, svc.Name)
```

Comment:
This commit needs a description of why you are removing this logic.

Reply:
Because "I think its useless" :1st_place_medal: ? 

No, in all seriousness: this is what I contacted you about on slack last week, and as I mentioned in the PR https://github.com/ovn-org/ovn-kubernetes/pull/978#issuecomment-632774827 I am not _sure_ it is useless and I would like to get your input on it. I have no problem dropping this commit, if you feel we should keep it, but I couldn't wrap my head around the logic for keeping it.

Reply:
> Because "I think it's useless"  ?
> 
> No, in all seriousness: this is what I contacted you about on slack last week, and as I mentioned in the PR [#978 (comment)](https://github.com/ovn-org/ovn-kubernetes/pull/978#issuecomment-632774827) I am not _sure_ it is useless and I would like to get your input on it. I have no problem dropping this commit, if you feel we should keep it, but I couldn't wrap my head around the logic for keeping it.

Reply:
OK, I'll branch out for that commit and create a separate PR